Modules/StudyProgramme/classes/class.ilObjStudyProgrammeGUI.php

```diff
@@ -190,6 +192,10 @@ public function executeCommand() {
 				$types_gui = new ilStudyProgrammeTypeGUI($this);
 				$this->ctrl->forwardCommand($types_gui);
 				break;
+			case 'ilobjectcopygui':
```

Comment:
Nein muss man nicht. Wenn man den Link zum kopieren "klaut", dann kommt zwar in den Ablauf, spätestes nach dem Klick auf "weiter" ist dieser aber kaputt, oder man bekommt eine Fehlermeldung.